cicecore/cicedyn/infrastructure/ice_domain.F90

```diff
@@ -160,6 +163,9 @@ subroutine init_domain_blocks
    maskhalo_remap    = .false.     ! if true, use masked halos for transport
    maskhalo_bound    = .false.     ! if true, use masked halos for bound_state
    halo_dynbundle    = .true.      ! if true, bundle halo updates in dynamics
+   sea_ice_time_bry  = .false.     ! if true, use time-varying sea-ice boundary files
```

Comment:
@SmithJos13 I think this is new namelist option that you introduced. Right? So, if you could use it to isolate your changes that would be great. So, another configuration will work without any issue.